tox.ini

```diff
@@ -16,13 +16,11 @@ install_command = pip install -e ".[test]" -U {opts} {packages}
 commands =
     py.test --cov=wagtailtrans --cov-report=xml tests/
 deps =
-    dj18: django>=1.8,<1.9
-    dj110: django>=1.10,<1.11
     dj111: django>=1.11,<2.0
-    dj18-mssql: django-pyodbc-azure==1.8.17.0
-    dj110-mssql: django-pyodbc-azure==1.10.4.0
     dj111-mssql: django-pyodbc-azure==1.11.0.0
-    wt112: wagtail>=1.12,<1.13
+    dj2: django>=2.0
+    dj2-mssql: django-pyodbc-azure==1.11.0.0
```

Comment:
This version-number is inline with the Django version. Right now it isn't available for 2.0 just yet. See: https://github.com/michiya/django-pyodbc-azure/issues/124